build-scripts/patches/moonray/apply

```diff
@@ -2,9 +2,12 @@
 
 DIR="$(realpath "$(dirname "${0}")")"
 
-# Configure git author
-git config user.email k8s-bot@canonical.com
-git config user.name k8s-bot
+# Configure git author if unset
```

Comment:
thank you for this!
I committed code more than once as `k8s-bot` because I applied moonray and forgot to change back the author.